data/campaigns/Winds_of_Fate/scenarios/02_The_Raid.cfg

```diff
@@ -0,0 +1,585 @@
+Better to do so before Karron's wing gets to him.
+"
+                condition=win
+                [show_if]
+                    [have_unit]
+                        id=Viragar
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Then destroy the intruders' remaining sea skimmers (boats) so they can not escape the hunt.
+Their sea skimmers are greatly vulnerable when moored at shore villages."
+                condition=win
+                [show_if]
+                    [have_unit]
+                        type=Outrigger
+                    [/have_unit]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Death of Viragar"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Reshan"
+                condition=lose
+            [/objective]
+            {TURNS_RUN_OUT}
+            [gold_carryover]
+                carryover_percentage=0
+            [/gold_carryover]
+            [note]
+                description= _ "Karron's Wing will attack from the northeast on the second dawn. (Turn 7)"
+            [/note]
+        [/objectives]
+    [/event]
+
+    [event]
+        name=start
+        [message]
+            speaker=Reshan
+            message= _ "Our swift approach escaped our quarry’s notice. Their defense is yet to be mustered."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "We will retake the nearest watchtower before they ready fully. From there we rally a defense of our own."
+        [/message]
+        [message]
+            speaker=Reshan
+            message= _ "If we are to ensnare their leader unharmed, then we must surround it with our hunters."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "Make that your first task, Reshan.
+
+Once we have our captive, burn their sea skimmers. The landbound prey may be left for our younger hunters to harvest."
+        [/message]
+    [/event]
+
+    [event]
+        name=attack end
+        [filter_second]
+            id=Grag
+        [/filter_second]
+        [message]
+            speaker=Grag
+            image_pos=right
+            mirror=yes
+            message= _ "These vomitting wyrms again!?  I HATE THEM!!!"
+        [/message]
+    [/event]
+
+    [event]
+        name=side 2 turn 1 refresh
+        [message]
+            speaker=Viragar
+            image_pos=right
+            mirror=yes
+            message= _ "Rally!  Rally!  Get to your posts, you scum! Wyrms are raining down on us!
+
+Pull our boats back and keep the fire breathers off of them!"
```

Comment:
Redundant 'of'. I don't think this is supposed to reflect the orcs' lower literary standards, should be 'keep the fire breathers off them'.

Reply:
It _is_ redundant, but it's a common redundancy, so I'm not sure how much this one matters…

Reply:
I don't think it's actually a mistake though; "off of them" sounds more correct to my ear than "off them" does.

Reply:
If it 'sounds' more correct to you that's probably because you hear it too often in your cultural context.

If it's not changed, then fine, I'll just fix it in the localisation.

Reply:
My point is that it's common enough to qualify as correct.

There could be an argument for it being a modern thing unsuited to the writing style of Wesnoth, though.

Reply:
It's possible it may just be a North American thing, too. I'm not sure. It doesn't sound wrong to my ear in any case.

Reply:
I will go with removing the "of" since that seems to fit the time period better.

Reply:
removed the "of"